common/src/test/java/org/apache/cassandra/sidecar/common/CassandraInputValidatorTest.java

```diff
@@ -133,11 +160,11 @@ public void testValidateFileName_withoutFileName_expectException()
     @Test
     public void testValidateSnapshotName_validSnapshotNames_expectNoException()
     {
-        ValidationUtils.validateSnapshotName("valid-snapshot-name");
-        ValidationUtils.validateSnapshotName("valid\\snapshot\\name"); // Is this really valid ??
-        ValidationUtils.validateSnapshotName("valid:snapshot:name");
-        ValidationUtils.validateSnapshotName("valid$snapshot$name");
-        ValidationUtils.validateSnapshotName("valid snapshot name");
+        instance.validateSnapshotName("valid-snapshot-name");
+        instance.validateSnapshotName("valid\\snapshot\\name"); // Is this really valid ??
```

Comment:
It is invalid for windows. For unix it is OK. Can we remove the question comment?